### PR TITLE
Support Haskell 'Infinity' as a Rescript float 'Infinity'

### PR DESCRIPTION
**Pull request by mchaver**

### CODE REVIEW

File: lib/js/__tests__/Aeson_decode_test.js [hunk 1]
```diff
@@ -130,6 +130,15 @@ Jest.describe("float", (function (param) {
         Jest.test("float", (function (param) {
                 return Jest.Expect.toEqual(Jest.Expect.expect(Aeson_decode.$$float(Aeson_encode.$$float(Number("-Infinity")))), Number("-Infinity"));
               }));
+        Jest.test("float", (function (param) {
```
Comment (mirko-plowtech) [inline lib/js/__tests__/Aeson_decode_test.js:133]: Screenshot(47)
^ I think only test for `+Infinity` was missing

**Review by mirko-plowtech — APPROVED** [review]
LGTM

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ## Changes
 
+### 5.1.3
```
Comment (mirko-plowtech) [inline Changelog.md:3]: 6.0.0 ? I mean the change is not retro compatible I think

Comment (shulhi) [reply Changelog.md:3]: It should be 6.1.0. Our 5.x branch is for older version of ReScript.

**Comment by shulhi** [comment]
@mchaver can you change the base branch to `rescript-v11` instead of `master`. The master currently for v10 support since Logan rollbacked the changes and I haven't actually merge the branch `rescript-v11` back to `master`

File: lib/js/__tests__/Aeson_decode_test.js (hunk 1 above)
Comment (mchaver) [reply lib/js/__tests__/Aeson_decode_test.js:133]: They are slightly different tests, one encodes a float directly, the other encodes a string then decodes a float:
```rescript
    test("float", () =>
      expect(Aeson.Decode.float(Aeson.Encode.float(Js.Float.fromString("Infinity"))))->toEqual(
        Js.Float.fromString("Infinity"),
      )
    )
    test("float", () =>
      expect(Aeson.Decode.float(Aeson.Encode.string("Infinity")))->toEqual(
        Js.Float.fromString("Infinity"),
      )
    )
```

**Comment by mchaver** [comment]
Replaced with another branch

**Pull request closed without merging**